workflows/dags/author/author_create/author_create_approved.py

```diff
@@ -48,43 +51,48 @@ def author_create_approved_dag():
     """
     inspire_http_hook = InspireHttpHook()
     inspire_http_record_management_hook = InspireHTTPRecordManagementHook()
-    workflow_management_hook = WorkflowManagementHook()
+    workflow_management_hook = WorkflowManagementHook(AUTHORS)
     workflow_ticket_management_hook = AuthorWorkflowTicketManagementHook()
 
     @task()
     def set_workflow_status_to_running(**context):
         status_name = "running"
         workflow_management_hook.set_workflow_status(
             status_name=status_name,
-            workflow_id=context["params"]["workflow_id"],
-            collection=AUTHORS,
+            workflow_id=context["params"]["workflow_id"]
         )
 
     @task.branch()
     def author_check_approval_branch(**context: dict) -> None:
-        """Branching for the workflow: based on create_ticket parameter
+        """Branching for the workflow: based on value parameter
         dag goes either to create_ticket_on_author_approval task or
         directly to create_author_on_inspire
         """
-        if context["params"]["create_ticket"]:
+        if "accept_curate" == context["params"]["data"]["value"]:
             return "create_author_create_curation_ticket"
         else:
-            return "empty_task"
+            return "close_author_create_user_ticket"
 
     @task
     def create_author_create_curation_ticket(**context: dict) -> None:
-        endpoint = "api/tickets/create"
-        request_data = {
-            "functional_category": "",
-            "workflow_id": context["params"]["workflow_id"],
-            "subject": "test",  # TODO: update subject and description
-            "description": "test",
-            "caller_email": "",  # leave empty
-            "template": "curation_needed_author",  # TODO: check template
-        }
-        response = inspire_http_hook.call_api(
-            endpoint=endpoint, data=request_data, method="POST"
-        )
+        
+        workflow_data = context["params"]["workflow"]["data"]
+        email = workflow_data["acquisition_source"]["email"]
+
+        bai = "[{}]".format(workflow_data.get("bai")) if workflow_data.get("bai") else ""
```

Comment:
why not f-string?

Reply:
i had copied it from next directly, my bad